### DISCUSSION TITLE
recommended way to document examples? Copy issue with >>> in code blocks, examples

### DISCUSSION THREAD

**Discussion by paoslaos** [squidfunk/mkdocs-material#6262] [General]
I was wondering what is the best way to document code examples such that they are nicely rendered in the docs later. I use pycharm and I think also generally, code examples are indicated with ">>>" in the doc string. This plays along with the code snippet / syntax highlighting such that the snippet can get copied. However, the ">>>" are also copied which makes copy & paste of example code a bit annoying. Is there a way to disable the ">>>" from being copied?  Omitting them is nicer for copy and paste but will make the output in the provided example to look like code. Any good solution for this? Just FYI: On GitHub it also does copy the ">>>".

Here a practical example of what I mean:

Typical way to document an example (at least to the best of my knowledge):

```python
>>> import numpy as np
>>> x = np.arange(0, 1)
array([0])
```

Alternative (?):

```python
import numpy as np
x = np.arange(0, 1)
array([0])
```

**Comment by alexvoss** [comment]
There has been a discussion about the more general question of excluding output - ah, here it is: #6252

**Reply by paoslaos** [reply]
Thanks! Of course i missed this, sorry!

**Reply by alexvoss** [reply]
You post is adding more context to this, which is helpful. I will add a link back to this to the earlier discussion (it does not seem to appear automagically?)
